### PR TITLE
html form added

### PR DESCRIPTION
**Pull request by NataliaSokolova**

### CODE REVIEW

File: index.html
Comment (brandaogabriel7) [inline index.html:1]: Run the "Format document" function in your code editor to make sure your indentation levels are consistent. You should always add one indent level for tags that are inside others.

File: js/index.js
Comment (brandaogabriel7) [inline js/index.js:1]: Your indent levels need some reviewing for this file, too.

**Review by brandaogabriel7 — APPROVED** [review]
Excellent job, Natalia!
Other than the indentation levels not being consistent in some places, everything looks great and works.

File: index.html
Comment (NataliaSokolova) [reply index.html:1]: Thank you, Gabriel! I run the function and it looks like formatted HTML.

**Pull request merged**